[PyTorch] add fastToString for infer_schema

swolchok opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #64691
* #64824
* __->__ #64823
* #64839

We seem to spend noticable time in vfprintf for this, and the number of arguments is almost always small enough to do this in just a few instructions.

Differential Revision: [D30860716](https://our.internmc.facebook.com/intern/diff/D30860716/)

swolchok committed:
[PyTorch] add fastToString for infer_schema

We seem to spend noticable time in vfprintf for this, and the number of arguments is almost always small enough to do this in just a few instructions.

Differential Revision: [D30860716](https://our.internmc.facebook.com/intern/diff/D30860716/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] add fastToString for infer_schema"


We seem to spend noticable time in vfprintf for this, and the number of arguments is almost always small enough to do this in just a few instructions.

Differential Revision: [D30860716](https://our.internmc.facebook.com/intern/diff/D30860716/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] add fastToString for infer_schema"


We seem to spend noticable time in vfprintf for this, and the number of arguments is almost always small enough to do this in just a few instructions.

Differential Revision: [D30860716](https://our.internmc.facebook.com/intern/diff/D30860716/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] add fastToString for infer_schema"


We seem to spend noticable time in vfprintf for this, and the number of arguments is almost always small enough to do this in just a few instructions.

Differential Revision: [D30860716](https://our.internmc.facebook.com/intern/diff/D30860716/)

[ghstack-poisoned]

ezyang commented:
This doesn't look semantics preserving; you're dropping the leading underscore when > 10

swolchok committed:
Update on "[PyTorch] add fastToString for infer_schema"


We seem to spend noticable time in vfprintf for this, and the number of arguments is almost always small enough to do this in just a few instructions.

Differential Revision: [D30860716](https://our.internmc.facebook.com/intern/diff/D30860716/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] add fastToString for infer_schema"


We seem to spend noticable time in vfprintf for this, and the number of arguments is almost always small enough to do this in just a few instructions.

Differential Revision: [D30860716](https://our.internmc.facebook.com/intern/diff/D30860716/)

[ghstack-poisoned]

swolchok committed:
Update on "[PyTorch] add fastToString for infer_schema"


We seem to spend noticable time in vfprintf for this, and the number of arguments is almost always small enough to do this in just a few instructions.

Differential Revision: [D30860716](https://our.internmc.facebook.com/intern/diff/D30860716/)

[ghstack-poisoned]
